src/integration_tests/schemas/e2e_schema_integration_test.py

```diff
@@ -17,43 +17,38 @@
 
 
 class E2ESchemaIntegrationTest(TestCase):
+    schema_metadatas = None
```

Comment:
typo?

Reply:
schema_metadatas is the response from /v1/schema_metadata and so is a list of metadata used in the for loop within the test methods